internal/rule/flatcase_test.go

```diff
@@ -0,0 +1,98 @@
+package rule
+
+import (
+	"testing"
+)
+
+func TestFlatcase(t *testing.T) {
+	rule := new(Flatcase).Init()
+
+	validTests := map[string]string{
+		"expected use case":                  "abc0123",
+		"lowercase letters":                  "abcdefghijklmnopqrstuvwxyz",
+		"lowercase letters with digits":      "0123456789",
+		"one lowercase letter":               "a",
+		"one digit":                          "1",
+		"lower case letters with diacritics": "åäöéèêçñ",
+		"lower case cyrillic letters":        "абвгдежзийклмнопрстуфхцчшщъыьэюя",
+		"lower case greek letters":           "αβγδεζηθικλμνξοπρστυφχψω", // Greek alpha lower case letter
+		"lower case armenian letters":        "աաբգդեէզէթժիյկլհմնոօփքռստուֆք",
```

Comment:
Could you please also add chinese letters - this was my intention in the past to support

Reply:
I will